src/modules/launcher/PowerLauncher.UI/LauncherControl.xaml

```diff
@@ -415,6 +415,21 @@
         Height="72" 
         Translation="0,0,16" 
         VerticalAlignment="Top">
+        <TextBox
+            x:Name="AutoCompleteTextBox"
+            x:FieldModifier="public"
+            Style="{StaticResource CustomAutoSuggestBoxTextBoxStyle}"
+            PlaceholderText="{Binding AutoCompleteText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
```

Comment:
I don't think we need twoway binding here.

Reply:
Sorry I missing something.  Where does AutoCompleteText get set?

Reply:
Thanks for pointing that out

Reply:
In MainViewModel.xaml.cs, lines 288 and 348.

Reply:
Actually you're right. I don't need the binding. Shall remove that. Sorry my bad